specification/ws.extensions/x-kaazing-ping-pong/pom.xml

```diff
@@ -0,0 +1,92 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.kaazing</groupId>
+        <artifactId>k3po.parent</artifactId>
+        <version>develop-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
```

Comment:
This s the wrong value for relativePath, that is why the builds are failing on Travis. relativePath needs to be `../../../pom.xml` (that is, one level higher, because this project is one level lower than the other projects under specifications).